crates/ra_ide_db/src/defs.rs

```diff
@@ -276,6 +276,14 @@ pub fn classify_name_ref(
         PathResolution::TypeParam(par) => Definition::TypeParam(par),
         PathResolution::Macro(def) => Definition::Macro(def),
         PathResolution::SelfType(impl_def) => Definition::SelfType(impl_def),
+        PathResolution::VariantDef(variant) => {
+            let def = match variant {
```

Comment:
we should just add an `impl From<VariantDef> for ModuleDef`.

Reply:
Yeah I agree this is cleaner in this case.